openpype/settings/lib.py

```diff
@@ -316,22 +316,6 @@ def _system_settings_backwards_compatible_conversion(studio_overrides):
                     }
 
 
-def _project_anatomy_backwards_compatible_conversion(project_anatomy):
-    # Backwards compatibility of node settings in Nuke 3.9.x - 3.10.0
-    # - source PR - https://github.com/pypeclub/OpenPype/pull/3143
-    value = project_anatomy
-    for key in ("imageio", "nuke", "nodes", "requiredNodes"):
-        if key not in value:
-            return
-        value = value[key]
-
-    for item in value:
-        for node in item.get("knobs") or []:
-            if "type" in node:
-                break
-            node["type"] = "__legacy__"
-
-
```

Comment:
Had no use with this move of the settings as far as I know - _so burned it down to the floor._

Reply:
https://github.com/pypeclub/OpenPype/pull/3909#discussion_r992079184